docs/REST API/rest-api.md

```diff
@@ -10760,7 +10760,7 @@ Returns the list of Apple App Store (VPP) that can be added to the specified tea
 
 _Available in Fleet Premium._
 
-Add App Store (VPP) app purchased in Apple Business Manager.
+Add Apple or Google App Store app. Apple apps must be added in Apple Business Manager (ABM) before adding them to Fleet.
```

Comment:
@marko-lisica the IT admin only needs to do something extra (add in ABM) for Apple apps, right?

IT admins can just add Google App Store apps to Fleet. There's no equivalent "Purchase licenses in ABM" step.